src/osd/PG.cc

```diff
@@ -980,6 +980,7 @@ void PG::remove_down_peer_info(const OSDMapRef osdmap)
       peer_missing.erase(p->first);
       peer_log_requested.erase(p->first);
       peer_missing_requested.erase(p->first);
+      peer_purged.insert(p->first); // so we can re-purge if necessary
```

Comment:
this should be ``peer_purged.erase()`` right?

Reply:
@liewegas Definitely. Just c&p from the purge_stray function :man_facepalming: 

Updated!